datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingListener.java

```diff
@@ -67,4 +67,9 @@
      * @return <code>true</code> if the permission was granted; <code>false</code> otherwise.
      */
     boolean onRequiresPermission(String permission, Reason reason);
+
+    /**
+     * This method is called when the capturing stopped.
```

Comment:
Maybe provide a little bit of explanation here on when this might occur. (Such as space is low).